Add Guidelines for Managing Follow-Up Issues (#12921)
kyma-project/test-infra: docs/how-to/how-to-manage-follow-up-issues.md

# Guidelines for Managing Follow-Up Issues

This document provides clear guidelines on how to effectively handle follow-up issues identified during events such as Technical Sprint Reviews.

## Rules

> 1. **Create Follow-Up Issues:**

Reviewer: I wouldn't use bold font in this way. Please have a look at the [guidelines for using bold font](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#items-written-in-bold-font) and adjust the whole doc.

Author: fixed

Always create a follow-up issue when it is identified and not resolved alongside the parent issue within the same Sprint.
2. **Team Discussion:**
Discuss the follow-up issue with the team during the Sprint Review or Sprint Planning sessions to ensure everyone is aware and aligned.

> 3. **Labeling and Closing Parent Issues:**
>    - **If the follow-up issue will not be added to the next Sprints:**
>      - Add a `technical-debt` label to indicate that this issue is a deferred concern.
>      - Close the parent issue to signify that its resolution is complete.
>      - **If the follow-up issue will be added to the next Sprints:**
>        - Do not add the `technical-debt` label, as it indicates a commitment to address the issue soon.
>        - Do not close the parent issue, as it remains open until the follow-up is resolved.

Reviewer: ```suggestion
3. **Labeling and Closing Parent Issues:**
   - **If the follow-up issue isn't added to the next Sprints:**
     - Add a `technical-debt` label to indicate that this issue is a deferred concern.
     - Close the parent issue to signify that its resolution is complete.
   - **If the follow-up issue is not added to the next Sprints:**
       - Do not add the `technical-debt` label, as it indicates a commitment to address the issue soon.
       - Do not close the parent issue, as it remains open until the follow-up is resolved.
```
